src/hw_5/t02_decorator.py

```diff
@@ -0,0 +1,62 @@
+""" Decorator task
+
+Create a decorator that stores results of all function calls (take into account
+all script executions).
+"""
+
+
+from os import path
+import random
+
+
+COLORS_LIST = ['red', 'green', 'blue', 'yellow', 'white']
+DICT_SEP = ':'
+ELEMENTS_SEP = ','
+FILE_NAME = 'calls.txt'
+RANDOM_RANGE = 100
+
+
+def set_call_result(f_name, f_result):
+    lines = []
+    function_found = False
+    if path.exists(FILE_NAME):
+        with open(FILE_NAME, 'r') as f:
+            for line in f:
+                function_name, res = line.split(DICT_SEP)
```

Comment:
you can use pickle module here (serialization/deserialization). But it's great. we can also add datetime.